cdk/src/main/java/dev/stratospheric/todoapp/cdk/ActiveMqApp.java

```diff
@@ -21,8 +21,8 @@ public static void main(final String[] args) {
     String region = (String) app.getNode().tryGetContext("region");
     Validations.requireNonEmpty(region, "context variable 'region' must not be null");
 
-    String username = (String) app.getNode().tryGetContext("username");
-    Validations.requireNonEmpty(username, "context variable 'username' must not be null");
+    String activeMqUsername = (String) app.getNode().tryGetContext("activeMqUsername");
```

Comment:
Hint: needs to be adjusted in the eBook sources

Reply:
this part of the `ActiveMqApp` is not part of the book -> no need to make changes